nonfree/README.md

```diff
@@ -0,0 +1,2 @@
+# gtsfm-nonfree
+3rd party algorithms to be plugged into GTSFM which are not free for commercial use due to licensing issues
```

Comment:
" due to licensing issues" is probably unnecessary.

Reply:
Thanks. removed it.